samples/smp_pi/src/main.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+
+/* Amount of execution threads to create and run */
+#define THREADS_NUM	16
```

Comment:
The sample output in the documentation uses different values for THREADS_NUM and DIGITS_NUM than shown here.